src/shared/views/MoreOptions.js

```diff
@@ -29,6 +28,7 @@ import {
     COLOR_RED_OVERLAY,
     COLOR_WHITE,
 } from '../constants';
+import Button from '../common/Button';
```

Comment:
Does this change have any impact on the visual aspect of the button for you? should we expect any differences? One of the difficult parts with the app is that each phone seems to have a slightly different screen size, which leads to designs potentially looking a bit different on various models.
RN is supposed to help with this, but I've often hard coded numbers like sizes and stuff, only to find out that it didn't work on such and such phones.

Reply:
No, I was quite happy when nothing (obvious?) changed once I adjusted the import